understanding/20/consistent-navigation.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Consistent Navigation</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Content can be navigated more predictably</dd>
-		<dt>Author task</dt><dd>Consistently order navigation that repeats across multiple pages</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Content can be navigated more predictably.</dd>
+         <dt>What to do</dt><dd>Consistently order navigation that repeats across multiple pages.</dd>
+         <dt>Why it's important</dt><dd>Content that behaves predictably is especially important to people with disabilities.</dd> 
```

Comment:
@GreggVan suggested
```suggestion
         <dt>Why it's important</dt><dd>It is much easier to understand how to navigate when there are consistent methods for doing so. Having different ways to do the same thing is confusing and makes it hard to predict or understand what will happen</dd> 
```

Reply:
This is almost the exact same problem I indentify in [my reply in consistent identification](https://github.com/w3c/wcag/pull/3356/files#r1330781028). Namely, a proposed line that is actually longer than the SC normative text is self-evidently not meeting the objective of being _brief_.

IMO, the existing wording works better.